src/data/polynomial/degree/definitions.lean

```diff
@@ -425,14 +425,8 @@ lemma nat_degree_mem_support_of_nonzero (H : p ≠ 0) : p.nat_degree ∈ p.suppo
 
 lemma nat_degree_eq_support_max' (h : p ≠ 0) :
   p.nat_degree = p.support.max' (nonempty_support_iff.mpr h) :=
-begin
-  apply le_antisymm,
-  { apply finset.le_max',
-    rw mem_support_iff_coeff_ne_zero,
-    exact (not_congr leading_coeff_eq_zero).mpr h, },
-  { apply max'_le,
-    refine le_nat_degree_of_mem_supp, },
-end
+le_antisymm (finset.le_max' _ _ $ nat_degree_mem_support_of_nonzero h) $
```

Comment:
Is the `finset` namespace already open here? If so,
```suggestion
le_antisymm (le_max' _ _ $ nat_degree_mem_support_of_nonzero h) $
```

Reply:
Or even
```suggestion
(le_max' _ _ $ nat_degree_mem_support_of_nonzero h).antisymm $
```

Reply:
Good catch, thanks! This model saw a lot more fully qualified names during training and it's easy to miss those.